test/humanizer.js

```diff
@@ -124,6 +124,20 @@ describe('humanizer', function () {
     assert.equal(h(2838550, { largest: 3 }), '47 minutes, 19 seconds')
   })
 
+  it('can do rounding with the "maxDecimalPoint" option', function () {
+    var h = humanizer({ maxDecimalPoints: 2 })
+
+    // assert.equal(h(8123.456789), '8.12 seconds')
```

Comment:
I think this is a valuable assertion, making sure that `maxDecimalPoints` isn't overridden.